cypress-tests/cypress/fixtures/bankTransfersConfirmBody/adyen-pix-confirm-body.json

```diff
@@ -0,0 +1,37 @@
+{
+  "confirm": true,
+  "return_url": "https://hyperswitch.io",
+  "payment_method": "bank_transfer",
+  "payment_method_type": "pix",
+  "payment_method_data": {
+    "bank_transfer": {
+      "pix": {}
+    }
+  },
```

Comment:
Why do we need a whole file for the fixture? Can't we reuse the confirm body and update the payment method data?

Reply:
Yes We can do that.
@pixincreate Can you make the changes for this